Integrations/ANYRUN/ANYRUN.py

```diff
@@ -71,7 +71,7 @@ def underscore_to_camel_case(s):
         The converted string (e.g. heLLoWorld).
     """
 
-    if not isinstance(s, STRING_TYPES):
+    if not isinstance(s, (str, bytes)):
```

Comment:
why did you change that?
the value of `STRING_TYPES` is set to that value incase of python3